app/Http/Requests/StoreNcpoRequest.php

```
@@ -0,0 +1,58 @@
+    public function rules(): array
+    {
+        return [
+            'date' => 'required|date',
+            'po_id' => 'required|exists:request_purchase_orders,id',
+            'justification' => 'required|string',
+            'approvals' => 'nullable|array',
+            'items' => 'required|array',
+            'items.*.item_id' => [
+                'required',
+                Rule::exists('item_profile', 'id')->where('is_approved', 1),
+                fn ($attr, $val, $fail) => !in_array($val, $this->validItems ?? [], true)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attr'. (undefined)

(UnusedFormalParameter)